SOURCE/FlowLaws.c

```diff
@@ -894,3 +894,30 @@ void ReadDataGSE( mat_prop* mat, params* model, int k, int number, scale* scalin
 /*--------------------------------------------------------------------------------------------------------------------*/
 /*------------------------------------------------------ M-Doodz -----------------------------------------------------*/
 /*--------------------------------------------------------------------------------------------------------------------*/
+
+void ReadDataKinetics( mat_prop* mat, params* model, int k, int number, scale* scaling ) {
```

Comment:
Number refers to phase_number? Phase_Id?

Reply:
bumber correspond to the ID of the kinetic law